### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by JmJames51**

### CODE REVIEW

**Comment by MarjorieThomassin** [comment]
conflicts to resolve

File: assets/styles/app.scss [hunk 1]
```diff
@@ -19,3 +26,34 @@ body {
...
+.landing-button-container{
+    text-align: center;
+}
+
+.landing-button{
+    background-color: $color-3;
+    font-size: 1em;
+    border-radius: 2rem;
+    width: 60vw;
+    height: 10vh;
```
Comment (rocambille) [inline assets/styles/app.scss]: buttons should not have a fixed size: use layout or spacing if you need it to take more space

File: templates/home/landingpage.html.twig [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+			<div>
+				<button class="landing-button">
+					S'inscrire
+				</button>
+			</div>
+			<div>
+				<button class="landing-button">
+					Se connecter
+				</button>
+			</div>
```
Comment (rocambille) [inline templates/home/landingpage.html.twig]: the div looks useless here... and your buttons will end up as links, so prefer <a> tags

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by MarjorieThomassin** [comment]
conflicts to resolve

**Pull request closed without merging**